prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh

```diff
@@ -163,13 +163,12 @@ fi
 CLEANUP_CLUSTER="true"
 (
 set -x
-kyma provision gardener \
-        --target-provider azure --secret "${GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME}" \
-        --name "${CLUSTER_NAME}" --project "${GARDENER_KYMA_PROW_PROJECT_NAME}" --credentials "${GARDENER_KYMA_PROW_KUBECONFIG}" \
-        --region "${GARDENER_REGION}" -t "${MACHINE_TYPE}" --disk-size 35 --disk-type=Standard_LRS --extra vnetcidr="10.250.0.0/16" \
+kyma provision gardener az \
+        --secret "${GARDENER_KYMA_PROW_PROVIDER_SECRET_NAME}" --name "${CLUSTER_NAME}" \
+        --project "${GARDENER_KYMA_PROW_PROJECT_NAME}" --credentials "${GARDENER_KYMA_PROW_KUBECONFIG}" \
+        --region "${GARDENER_REGION}" -t "${MACHINE_TYPE}" --disk-size 35 \
         --nodes 4 --scaler-min 3 \
```

Comment:
Why you removed the zone here but not for aws? Why here is a custom disk size but not for aws?

Reply:
I made it all same.